doc/notes/plutus-core/Builtins_Overview.md

```diff
@@ -0,0 +1,350 @@
+# An Overview of Built-in Types and Functions
+
+The Plutus ledger languages are parameterised by the built-in types and 
+built-in functions they supports.
+
+## Built-In Versions
+
+A built-in version is a fixed set of built-in types and built-in
+functions. Each Plutus ledger language supports a particular built-in
+version.
+
+Therefore, a built-in with the same name may or may not exist in
+different versions and its type and implementation might change across
+versions.
+
+Hence, it is useful to think of a built-in as being indexed by its
+version. We may write this as a built-in name with its version as a
+subscript. For example, the built-in function `ifThenElse` at
+version 1.1 may be written as `IfThenElse_1.1`. For a formalisation,
+having the version as an index, rather than a parameter, fits better
+with the fact that a built-in might only exist at a certain version.
+
+As a convention, the version may be left implicit. When the version is left 
+implicit, the version should be clear from the context. Without the version 
+we cannot talk about a built-in's existence, type, or denotation.
+
+    * In the implementation the version is a
```

Comment:
This doesn't render correctly - check the rich diff to see.

Reply:
Right. Fixed it.